llvm/test/CodeGen/PowerPC/uaddo-32.ll

```diff
@@ -0,0 +1,38 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=powerpc-unknown-linux-gnu | FileCheck %s
+; RUN: llc < %s -mtriple=powerpc-ibm-aix-xcoff | FileCheck %s
+
```

Comment:
Are these IR not valid for 64bit?  I would think we can just have 1 tc and make sure we test for both 32 and 64bit on Linux and AIX.